src/connectors/joox.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+
+
+const defaultAlbumData = {
+	path: undefined,
+	loaded: false,
+	name: undefined
+};
+
+const playerSelector = '#jooxPlayerBar';
```

Comment:
I'd get rid of these constants and assign selector strings to `Connector.%someProperty%` directly.